balena/models/device_os.py

```diff
@@ -259,12 +259,13 @@ def get_device_os_semver_with_variant(self, os_version, os_variant=None):
             xstr(version_info.build) + '.' + os_variant if os_variant and os_variant not in tmp else version_info.build
         )
 
-    def get_supported_versions(self, device_type):
+    def get_supported_versions(self, device_type, auth=None):
         """
         Get OS supported versions.
 
         Args:
-            device_type (str): device type slug
+            device_type (str): device type slug.
+            auth (Optional[bool]): if auth is True then auth header will be added to the request (to get private device types) otherwise no auth header and only public device types returned. Default to None.
```

Comment:
Should it now say that default/None would use the bearer token if the sdk instance is logged in?